src/plugins/hetero/plugin.cpp

```diff
@@ -18,7 +18,7 @@
 #include "cpp_interfaces/interface/ie_internal_plugin_config.hpp"
```

Comment:
Looks like you need to remove this file (the same for executable network).
Please remove legacy files and change cmake scripts if it is needed